roles/pulp3-devel/tasks/main.yml

```diff
@@ -35,11 +34,10 @@
           - python-django-bash-completion
           - python3-setuptools
           - python3-virtualenvwrapper
-          - redhat-lsb-core
           - ripgrep
           - vim-enhanced
         state: present
-      retries: "{{ package_retries }}"
+      retries: "{{ pulp_devel_package_retries }}"
```

Comment:
This isn't defined properly as Travis is erroring out still seeking the old package_retries.
""The field 'retries' has an invalid value, which includes an undefined variable. The error was: 'package_retries' is undefined"